src/components/Value/index.tsx

```diff
@@ -28,10 +28,10 @@ export const ValueSection = ({ values, ...sectionProps }: IValueSection) => {
       <Container title="Our Values" className="border-b px-0 pb-20">
         <div className="container flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
           {values.map((value, key) => (
-            <div className="flex flex-col w-100 sm:w-100 sm:items-center sm:text-center pr-16 sm:pr-0">
+            <div className="flex flex-col text-center w-100 sm:w-100 sm:items-center sm:text-center pr-16 sm:pr-0">
```

Comment:
I suggest doing something like "mx-8" instead of a right padding. If you do right padding only, it you will end up with no padding to the left of the left most element and a bunch of padding to the right of the last element. Notice the alignment with the centered text:

**Before**

![image](https://user-images.githubusercontent.com/7423098/71368456-9df58e00-256d-11ea-97f0-9a182847cab0.png)


**After**

![image](https://user-images.githubusercontent.com/7423098/71368429-8c13eb00-256d-11ea-9376-02483ae06e6a.png)


```suggestion
            <div className="flex flex-col text-center w-100 sm:w-100 sm:items-center sm:text-center mx-8">
```